src/components/Search.js

```diff
@@ -168,6 +173,7 @@ class Search extends Component {
 
             this.setState({
                 preferences: preferences.message.default,
+                customMessage: preferences.message.customMessage || {},
```

Comment:
@jyash97 There might be a case when the `preferences.message` can be `undefined` which will crash the app.
So it's better to use `get` from lodash to avoid such situations.